mobile-app/app/screens/AppNavigator/screens/Loans/components/CollateralizationRatioDisplayV2.tsx

```diff
@@ -16,6 +16,8 @@ interface CollateralizationRatioDisplayProps {
   totalLoanAmount: string;
   testID: string;
   showProgressBar?: boolean;
+  isReadyForLoan?: boolean;
```

Comment:
question: how do you decide this?

Reply:
for add collateral, when vault has no loan then's it's ready for loan

Reply:
can we use collateralAmounts for that? actually we can also change collateralAmounts to `totalCollateralValue`

Reply:
Yes if we're going to use `totalCollateralValue` instead of `collateralAmounts` then I won't need this extra `isReadyForLoan` flag